aries-site/src/pages/components/skeleton-loading.mdx

```diff
@@ -0,0 +1,33 @@
+import { Notification } from 'grommet';
+import { Example } from '../../layouts';
+import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
+
+<Example
+  componentName="Skeleton"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js"
+  docs="https://v2.grommet.io/skeleton?theme=hpe#props"
+>
+  <DashBoardSkeletonExample />
+</Example>
+
+### When to use
+Skeletons are used when:
```

Comment:
Not sure where this would go (probably elsewhere in the gudiance) we should mention that data should be rendered as soon as it's fetched. Using Instagram as an example, there may be multiple fetches happening (1 for stories and another for the main feed). If the stories response comes back before the main feed one, we should render the stories right away and just leave the main feed as a skeleton).

With how grommet skeleton is built, this is easy to implement by just controlling what container you flag the `skeleton` prop on. Might be a nice opportunity to highlight the ease of implementing with Grommet.

Reply:
agree this was some feedback we got from Eric I think were going to include just not sure where but noted ✅

Reply:
Cool sounds good.